modules/features/player/src/main/res/layout/fragment_player_container.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<au.com.shiftyjelly.pocketcasts.player.helper.NestedCoordinatorLayout
```

Comment:
As far as I could tell and test `NestedCoordinatorLayout` didn't help with anything anymore. It was added in the [mentioned PR](https://github.com/Automattic/pocket-casts-android-private/pull/1862) that was helping with dragging issues but I did not experience any.